### PR TITLE
[WOR-1581] Thread billing profile organization through Rawls billing project response

### PR DESCRIPTION
**Pull request by marctalbott**
Ticket: [WOR-1581](https://broadworkbench.atlassian.net/browse/WOR-1581)
* Include billing profile organization information in Rawls billing project response if present in BPM's response

---

**PR checklist**

- [x] Include the JIRA issue number in the PR description and title
- [x] Make sure Swagger is updated if API changes
  - [ ] **...and Orchestration's Swagger too!**
- [ ] If you changed anything in `model/`, then you should [publish a new official `rawls-model`](https://github.com/broadinstitute/rawls/blob/develop/README.md#publish-rawls-model) and update `rawls-model` in [Orchestration's dependencies](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/project/Dependencies.scala).
- [ ] Get two thumbsworth of PR review
- [ ] Verify all tests go green, including CI tests
- [ ] **Squash commits and merge** to develop (branches are automatically deleted after merging)
- [ ] Inform other teams of any substantial changes via Slack and/or email

[WOR-1581]: https://broadworkbench.atlassian.net/browse/WOR-1581?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by aherbst-broad — APPROVED** [review]
lgtm; be sure to update release notes with a brief description of the API change.

File: core/src/main/resources/swagger/api-docs.yaml [hunk 1]
```diff
@@ -6515,6 +6515,8 @@ components:
         protectedData:
           type: boolean
           description: whether this billing project supports protected data (cloudPlatform AZURE only)
+        organization:
+          $ref: '#/components/schemas/RawlsBillingProjectOrganization'
```
Comment (cahrens) [inline core/src/main/resources/swagger/api-docs.yaml:6519]: Remember to update orchestration's Swagger.

File: core/src/main/resources/swagger/api-docs.yaml [hunk 2]
```diff
@@ -6533,6 +6535,15 @@ components:
...
           description: ID of the Azure managed resource group for this managed application deployment.
+    RawlsBillingProjectOrganization:
+      required:
+        - enterprise
+      type: object
+      description: details about billing project's organization
+      properties:
+        enterprise:
+          type: boolean
+          description: whether this billing project is part of an enterprise organization i.e. they have a signed BAA
```
Comment (cahrens) [inline core/src/main/resources/swagger/api-docs.yaml]: Instead of BAA (or in additionto), can you put whatever it means? I can't even remember….

Comment (cahrens) [reply core/src/main/resources/swagger/api-docs.yaml]: If `organization` will not always be present, please make a note of that.

**Review by cahrens — APPROVED** [review]
Just some documentation suggestions.

File: core/src/main/resources/swagger/api-docs.yaml (hunk 1 above)
Comment (marctalbott) [reply core/src/main/resources/swagger/api-docs.yaml:6519]: planning to publish the new model changes and update Orch's swagger in the process

**Pull request merged**